ui/components/app/confirm/info/row/text.stories.tsx

```diff
@@ -22,4 +22,12 @@ DefaultStory.args = {
   text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
 };
 
+export const EditableStory = ({ text, onEditClick }) => (
+  <ConfirmInfoRowText text={text} onEditClick={onEditClick} />
+);
+EditableStory.args = {
+  text: 'Lorem ipsum dolor sit amet.',
+  onEditClick: () => console.log('Edit button clicked!'),
```

Comment:
non-blocking: if you like to do this the storybook way you could include this prop and assign it an action

```jsx
import React from 'react';
import { ConfirmInfoRow } from './row';
import { ConfirmInfoRowText } from './text';
import { on } from 'events';

const ConfirmInfoRowTextStory = {
  title: 'Components/App/Confirm/InfoRowText',
  component: ConfirmInfoRowText,
  decorators: [
    (story) => <ConfirmInfoRow label="Message">{story()}</ConfirmInfoRow>,
  ],
  argTypes: {
    url: {
      control: 'text',
    },
    onEditClick: {
      action: 'onEditClick',
    },
  },
};

export const EditableStory = (args) => <ConfirmInfoRowText {...args} />;
EditableStory.args = {
  text: 'Lorem ipsum dolor sit amet.',
};

export default ConfirmInfoRowTextStory;

```


https://github.com/MetaMask/metamask-extension/assets/8112138/e4b0d29f-b515-4bc5-9164-194c9766d27b

Reply:
Thanks for the suggestion, included